documentation/projects/proposals/nuxt_3_migration/20230608-implementation_plan_nuxt_3_migration.md

```diff
@@ -0,0 +1,277 @@
+| `"@nuxtjs/proxy"`                  | Replace with Nitro routeRules or use `proxyRequest`                                                  | https://stackoverflow.com/questions/76120894/how-do-i-set-proxy-with-baseurl-with-nuxt3-and-nitro-config, https://github.com/nuxt/nuxt/discussions/15907 |
+| `"@nuxtjs/redirect-module"`        | Define Nitro routeRules in `nuxt.config`                                                             | https://nitro.unjs.io/config/#routerules                                                                                                                 |
+| `"@nuxtjs/sentry"`                 | No official support                                                                                  | Workaround in https://github.com/nuxt-community/sentry-module/issues/530                                                                                 |
+| `"vue-plausible"`                  | Update to current version that supports Nuxt 3 or https://github.com/nuxt-modules/plausible          | Note: enabling outbound tracking breaks links with target blank in `vue-plausible`                                                                       |
+| `"@nuxtjs/sitemap"`                | Replace with `nuxt-simple-sitemap`                                                                   | https://github.com/harlan-zw/nuxt-simple-sitemap                                                                                                         |
+
+We should probably add [`nuxt-vitest`](https://github.com/danielroe/nuxt-vitest)
+for testing in Nuxt. The package is under active development, so we should pin
+to the patch version
+
+The migration to Nuxt 3 also means migration from Vue 2 (with added composition
+API) to Vue 3. Below is a list of breaking changes that need to be addressed in
+Openverse.
+
+#### Vue 3 breaking changes
+
+- [ ] `v-model`
+      [changes](https://v3-migration.vuejs.org/breaking-changes/v-model.html)
+- [ ] `key` usage on `template v-for` instead of the components inside that
+      `template`
+- [ ] [`v-on:event.native` removed](https://v3-migration.vuejs.org/breaking-changes/v-on-native-modifier-removed.html) -
+      after removing `.native`, check that links work as expected, sending the
+      analytics events
+- [ ] [`$listeners` has been removed / merged into `$attrs`](https://v3-migration.vuejs.org/breaking-changes/listeners-removed)-
+      remove the `$listeners` everywhere and make sure that we have
+      `v-bind="$attrs"` everywhere we had `$listeners` before
+- [ ] [`$attrs` includes `class` & `style`](https://v3-migration.vuejs.org/breaking-changes/attrs-includes-class-style.html) -
+      remove `v-bind="$attrs"` from the root element in components where
+      `inheritAttrs="false"`
```

Comment:
Can you give more details for why this is the modification required to handle this breaking change? I'm not making the connection between `inheritAttrs="false"` and the class and style attributes being included in `$attrs`.